### PR TITLE
ENH announce master to main migration

### PR DESCRIPTION
**Pull request by beckermr**
PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

**Comment by beckermr** [comment]
cc @conda-forge/core

File: src/user/announcements.rst [hunk 1]
```diff
@@ -5,6 +5,16 @@ Announcements
 
 Our announcements are published to an RSS feed `here <https://conda-forge.org/docs/news.rss>`_.
 
+2022
+----
+
+:2022-02-13: Default branch migration from ``master`` to ``main``
+
+    We will be migrating the default branches of all feedstocks and other ``conda-forge``
```
Comment (beckermr) [inline src/user/announcements.rst:13]:
```suggestion
    We will be migrating the default branches of all feedstocks and other ``conda-forge`` repos
```

File: src/user/announcements.rst [hunk 2]
```diff
@@ -5,6 +5,16 @@ Announcements
...
+    from ``master`` to ``main``. We do expect some minor hiccups while this migration is 
+    going on. If you encounter any problems, please comment on this Github 
+    `issue <https://github.com/conda-forge/conda-forge.github.io/issues/1162>`__.
```
Comment (beckermr) [inline src/user/announcements.rst:16]:
```suggestion
    _`issue <https://github.com/conda-forge/conda-forge.github.io/issues/1162>`__.
```

**Comment by hmaarrfk** [comment]
Exciting!

**Pull request merged**